### DISCUSSION TITLE
Unable to install on Debian 12 Bookworm RPi

### DISCUSSION THREAD

**Discussion by k5map** [fastfetch-cli/fastfetch#1079] [Q&A] [answered]
On a freshly built RPi4, when I type "sudo apt install fastfetch", receive the response "E: Unable to locate package fastfetch".  Is there an alternate way to install it?

**Comment by CarterLi** [comment, marked as answer]
You may try <https://github.com/fastfetch-cli/fastfetch/releases/download/2.17.2/fastfetch-linux-aarch64.deb>

**Reply by k5map** [reply]
@CarterLi ... thanks for the link.  That worked!

**Comment by Guiorgy** [comment]
@k5map fastfetch is currently not available in the `main`/`stable` repos yet, it has however recently been added to the `testing` and `unstable` repos (see [here](https://packages.debian.org/search?keywords=fastfetch)). If you want to be able to install and ubdate it through `apt`, concider using the [unstable](https://wiki.debian.org/DebianUnstable) repo.

**Reply by KaKi87** [reply]
To get automated updates on stable Debian 12, adding the PPA then replacing `bookworm` with `jammy` also works. That said, official availability would be nice. Thanks

*(Prerequisite : the `software-properties-common` package providing the `add-apt-repository` command)*.
